### ISSUE TITLE
Colour contrast in the Citation section is not ideal

### ISSUE THREAD

**Issue by hturner** [rjournal/rjtools#96]
Picked up with Accessible Web Helper extension:

Colour contrast in the Citation section fails check "Ensures the contrast between foreground and background colors meets WCAG 2 AA minimum contrast ratio thresholds."

**Comment by dicook** [comment]
Related to https://github.com/rjournal/rjtools/issues/95#issue-1888038156
